### PR TITLE
Data member

### PR DESCRIPTION
**Pull request by kain88-de**
my first clumsy try of introducing the data limb as a member of the Sim object.

### CODE REVIEW

File: src/mdsynthesis/data.py [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+        in which data appears, goes missing, etc. while a Treant is loaded.
+
+        .. note:: Methods wrapped with this decorator need to have *handle* as
+                  the first argument.
+
+        """
+
+        @wraps(func)
+        def inner(self, handle, *args, **kwargs):
+            dirname = os.path.join(self.treant.abspath, handle)
```
Comment (kain88-de) [inline src/mdsynthesis/data.py:131]: @dotsdl  This change enables to run the tests without an explicit error being thrown. But now there is an infinite recursion happening saving a dataframe.

**Comment by kain88-de** [comment]
@dotsdl I got this working on my laptop now. This looks like a very good step to get mdsynthesis running in time with a new datreant release. I'd appreciate if you have a look over this.

**Comment by kain88-de** [comment]
Also ping @richardjgowers

File: src/mdsynthesis/data.py (hunk 1 above)
Comment (kain88-de) [reply src/mdsynthesis/data.py:131]: This was due to a missing pytables package. Still a weird error at least to me.

**Comment by kain88-de** [comment]
@dotsdl  ping

**Comment by kain88-de** [comment]
@dotsdl  ping

**Comment by dotsdl** [comment]
Reviewing now. Thanks @kain88-de!

File: src/mdsynthesis/treants.py [hunk 2]
```diff
@@ -59,6 +63,10 @@ def __init__(self, sim, categories=None, tags=None):
         # make simdir
         self._make_simdir()
 
+        # make simdir
+        self._make_simdir()
```
Comment (dotsdl) [inline src/mdsynthesis/treants.py]: Duplicate init using `_make_simdir()` necessary here?

File: src/mdsynthesis/treants.py [hunk 3]
```diff
@@ -2,8 +2,11 @@
 Basic Treant objects: the organizational units for :mod:`mdsynthesis`.
 
 """
+from __future__ import absolute_import
```
Comment (dotsdl) [inline src/mdsynthesis/treants.py:5]: What does `absolute_import` give us in this module?

**Review by dotsdl — COMMENTED** [review]
Looks good! I'm trying to get the tests running against current `datreant` development branch. Mind if I push changes to make that work?

**Comment by kain88-de** [comment]
@dotsdl I restarted the build. This will pull the latest changes of the datreant branch.

File: src/mdsynthesis/treants.py (hunk 3 above)
Comment (kain88-de) [reply src/mdsynthesis/treants.py:5]: enforces that we need to do `from . import ...` for packae internal files under python2.7. It's a common idiom for python2/3 compatibility.

File: src/mdsynthesis/treants.py (hunk 2 above)
Comment (kain88-de) [reply src/mdsynthesis/treants.py]: I wouldn't think so

**Comment by dotsdl** [comment]
Cool! Looks like this passes. Merge ahead when you like!

**Pull request merged**